### ISSUE TITLE
wiki doesn't show up in help

### ISSUE THREAD

**Issue by halkeye** [halkeye/hubot-confluence-search#2]

**Comment by bisscuitt** [comment]
This appears to be working now (atleast from my dev shell):

```
devbot> devbot help wiki
devbot wiki <term> - Search term to look up
```

Also works on my production hubot instance running with the Slack adapter.

**Comment by halkeye** [comment]
Woo! someone else is using it!

But yea, it was specifically fixed in 4f049b76ff1545bcf0fd6cfb4a2886c0216c2d49 but I forgot to close the issue :)
